crates/poolnhl_app/src/main.rs

```diff
@@ -27,6 +28,11 @@ async fn main() {
             .expect("Was not able to query the JWKS from hanko server."),
     );
     let services = ServiceRegistry::new(db, cached_jwks);
+    services
+        .pool_service
+        .init_indexes()
```

Comment:
❓ Is that a `SQL` index? Do you really want to create that every time the service starts or only if the database is not created?